resources/assets/js/components/importer/importer.vue

```diff
@@ -63,7 +63,7 @@
 
         methods: {
             fetchFiles() {
-                this.$http.get(route('api.imports.index'))
+                this.$http.get('api/v1/imports')
```

Comment:
Same here (and all the rest) - we need to prepend the actual app url to all of these or GH will light up like a very sad, ugly-crying christmas tree

Reply:
Yes. Do not want.